### ISSUE TITLE
Switching to windows theme will cause window to hang [v2.2.0]

### ISSUE THREAD

**Issue by de0xy** [lwouis/alt-tab-macos#109]
### Error message:
```
Could not save Preference
Optional contained nil
[Check again] [Cancel] [Edit]
```

**Comment by lwouis** [comment]
Thanks a lot for reporting the issue. This is a recent regression introduced 2 days ago in 51a193c1d43ad07205b4b24ed48ada8e114212f7. I just merged #112 to fix it.

@de0xy Could you please try [release 2.3.1](https://github.com/lwouis/alt-tab-macos/releases/tag/v2.3.1)? It should address the issue :)

**Comment by de0xy** [comment]
Thanks, looks like it fixed the issue 👍
